src/svg/logo/buildit-logotype.svg

```diff
@@ -1,79 +1,13 @@
+    </g>
+    <path d="M103.506 1.598l.047 25.35" stroke="#0060FF" stroke-width="2.117" stroke-linecap="square"/>
+    <g fill="#0060FF" fill-rule="nonzero">
```

Comment:
Can you please add `class="grav-svg-hl-1"` attributes to the vertical line and "buildit" word?

That allows us to recolour that portion of the graphic using CSS in the UI lib.